activerecord/test/cases/adapters/postgresql/uuid_test.rb

```diff
@@ -195,17 +210,52 @@ def test_pk_and_sequence_for_uuid_primary_key
       assert_equal nil, seq
     end
 
-    def test_schema_dumper_for_uuid_primary_key
-      schema = dump_table_schema "pg_uuids"
-      assert_match(/\bcreate_table "pg_uuids", id: :uuid, default: -> { "uuid_generate_v1\(\)" }/, schema)
-      assert_match(/t\.uuid   "other_uuid", default: -> { "uuid_generate_v4\(\)" }/, schema)
+    if ActiveRecord::Base.connection.supports_pgcrypto_uuid?
+      def test_schema_dumper_for_uuid_primary_key
+        schema = dump_table_schema "pg_uuids"
+        assert_match(/\bcreate_table "pg_uuids", id: :uuid, default: -> { "uuid_generate_v1\(\)" }/, schema)
+        assert_match(/t\.uuid   "other_uuid", default: -> { "uuid_generate_v4\(\)" }/, schema)
+      end
+    else
+      def test_schema_dumper_for_uuid_primary_key
+        schema = dump_table_schema "pg_uuids"
+        assert_match(/\bcreate_table "pg_uuids", id: :uuid, default: -> { "uuid_generate_v4\(\)" }/, schema)
```

Comment:
Two different columns. The `id` column, also the primary key, uses the `uuid_generate_v1()` function while `other_uuid` uses `uuid_generate_v4()`. Both are functions in the `uuid-ossp` extension.